support-frontend/app/lib/RedirectWithEncodedQueryString.scala

```diff
@@ -0,0 +1,14 @@
+package lib
+
+import play.api.http.Status
+import play.api.mvc.{Result, Results}
+
+object RedirectWithEncodedQueryString extends Results with Status {
+  //Encode the querystring parameter keys, as Play only encodes the values
+  def apply(url: String, queryString: Map[String, Seq[String]] = Map.empty, status: Int = SEE_OTHER): Result = Redirect(
+    url = url,
```

Comment:
@tomrf1 what do you think of this change for safety/defensiveness? Is there any risk?
```suggestion
    url = java.net.URLEncoder.encode(url, "utf-8"),
```

Reply:
good question.
Based on [this answer](https://stackoverflow.com/a/7654605), you are correct that it should be encoded in the `Redirect` header.
Please test that redirects still work with the change

Reply:
This change seems to break the redirect for contributions so I'm going to bin it